modules/nw-udn-limitations.adoc

```diff
@@ -0,0 +1,19 @@
+//module included in the following assembly:
+//
+// *networkking/multiple_networks/understanding-user-defined-networks.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="limitations-for-udn_{context}"]
+= Limitations for UserDefinedNetwork custom resource
+
+While user-defined networks (UDN) offer highly customizable network configuration options, there are limitations that cluster administrators and developers should be aware of when implementing and managing these networks. Consider the following limitations before implementing a user-defined network.
+
+* *Scale limitations*: The number of internal OVN-Kubernetes objects, such as kernel devices, rules, and VRFs, multiply as the number of namespaces and networks increase. As a result, user-defined networks might cause potential performance issues and scalability challenges for some {product-title} deployments.
+* *Complex network configurations*: User-defined networks reduce the complexity of isolation policies. However, setting up and managing multiple networking, wherein each have their own gateway router and load balancer, requires advanced networking knowledge.
+* *Interference with secondary projects*: User-defined networks increase the risk of breaking secondary projects that integrate with OVN-Kubernetes, such as MetalLB or Submariner.
+* *DNS limitations*: DNS lookups for pods resolve to the pod's IP address on the cluster default network. Even if a pod is part of a user-defined network, DNS lookups will not resolve to the pod's IP address on that user-defined network.
+* *Initial network assignment*: If a namespace is assigned to a new network, it cannot have existing pods, or else the new network assignment will not be accepted by OVN-Kubernetes.
+* *Service reachability*: Services created in namespaces that are served by the UDN are only accessible by namespaces connected to the UDN. Services in a UDN are reachable by other namespaces that share the same network. This can limit the flexibility of services across different networks.
+
+* *Health check limitations*: Kubelet health checks are performed by the cluster default network, which does not confirm the network connectivity of the primary interface on the pod. Consequently, scenarios where a pod appears healthy by the default network, but has broken connectivity on the primary interface, are possible with user-defined networks.
```

Comment:
I think it is appropriate to call this out. I can imagine a scenario of an app that has a  hard-coded `eth0` as the interface some server is listening on, this could be deceiving for the end user.

Reply:
Ok let's keep it then. I forgot this set is coming from the corner cases of UDN docs.. so maybe it's ok to leave them

Although Joe all these are not really Best practices... Maybe some of these are implementation details we want to inform the end users about..

Reply:
These are all in limitations.